### ISSUE TITLE
Can't use in swift 5 Xcode 10.3

### ISSUE THREAD

**Issue by changanli** [CocoaLumberjack/CocoaLumberjack#1092]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/CocoaLumberjack/CocoaLumberjack/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](https://github.com/CocoaLumberjack/CocoaLumberjack#documentation) or [CocoaDocs](cocoadocs.org/docsets/CocoaLumberjack)
* [ ] I have searched for a similar issue in the [project](https://github.com/CocoaLumberjack/CocoaLumberjack/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / osx / tvos / watchos
 Platform Version        | e.g. 8.0
 CocoaLumberjack Version | ~ 3.5.3
 Integration Method      | manually
 Xcode Version           | Xcode 10.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Running `DDLogDebug("测试一下");` does not print anything.

**Comment by ffried** [comment]
@changanli Could you please fill out the issue template and provide a little more details on what is not working?
Also, there is no Swift 5.2 yet. There's Swift 5.0 (which is the current stable version) and there's Swift 5.1 (which is currently in beta and available with Xcode 10.4 beta). Since we don't officially support beta's, you might see errors there.
Unofficially, however, I can tell you that CocoaLumberjack should currently work with Swift 5.1. 😉

**Comment by changanli** [comment]
can't not print any thing!!! I don't know why!!!

**Comment by ffried** [comment]
@changanli This does not answer much of what I asked.
Again:
- What Swift version are you referring to (there is no Swift 5.2)?
- What version of CocoaLumberjack are you using?
- What integration method are you using (CocoaPods, Carthage, manual)?
- Have you tried to reproduce this in a small sample project?

A few more that could help narrow this down:
- What loggers are you using?
- What's the level of the messages you're trying to log?
- Have you changed the global log level (and if so, to what)?

**Comment by changanli** [comment]
Xcode 10.3 swift_version: Swift 5 
maybe 3.5.3, i download today
integration: manual
DDLogDebug("测试一下");

**Comment by ffried** [comment]
@changanli Thanks. I've updated the issue description based on your reply.

`DDLogDebug` is the second most verbose way to print a message. It should not be filtered by default, but could you please test logging your message using `DDLogError`? By default, `DDLogError` logs synchronously and is rarely filtered.

If that doesn't work, then I still need to know what loggers you're using.

**Comment by changanli** [comment]
it dosen't work。
 DispatchQueue.main.asyncAfter(deadline: .now() + 0.2) {
                DDLogError("测试一下")
            }

**Comment by changanli** [comment]
Demo:  https://github.com/changanli/CocoaLumberjackTest
Xcode 10 swift 5 , you can download this,Thank you!

**Comment by ffried** [comment]
@changanli I've had a look at your project. You never add any loggers and thus no messages will be logged. Please refer to the "How to get started" section in our [README](https://github.com/CocoaLumberjack/CocoaLumberjack/blob/master/README.md) to see an example on how to add loggers.

The `DDLogXYZ` functions just pass the message to a `DDLog` instance (if it passes the global log level). `DDLog` will then dispatch the message to all added loggers. In your case no logger was ever added and therefor the message(s) will simply be dropped.
